examples/tca_example/.bazelignore

```diff
@@ -0,0 +1 @@
+.build
```

Comment:
swift-syntax defines BUILD files so this is needed to prevent `do_test` from finding it and failing with this error:

```
ERROR: error loading package under directory '': error loading package '.build/checkouts/swift-syntax': Label '//utils/bazel:swift_syntax_library.bzl' is invalid because 'utils/bazel' is not a package; perhaps you meant to put the colon here: '//:utils/bazel/swift_syntax_library.bzl'?
```